spec/features/create_preassembly_image_spec.rb

```diff
@@ -129,8 +131,36 @@
     yaml = YAML.load_file(download)
     expect(yaml[:status]).to eq 'success'
 
-    visit "#{Settings.argo_url}/view/druid:#{yaml[:pid]}"
+    prefixed_druid = "druid:#{yaml[:pid]}"
```

Comment:
maybe add a comment before this section to indicate this is all about confirming preservation replication to could endpoints is working